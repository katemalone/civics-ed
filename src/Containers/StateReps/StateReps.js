import React from 'react'; 

export const StateReps = ({ name, image, email, party, chamber }) => {
    return(
      <section className="StateReps_card">
        <h3>{name}</h3>
        <p>Chamber: {chamber}</p>
        <p>{party}</p>
        <p>Contact info {email}</p>
        <img src={image} />
      </section>
    )
}

export default StateReps;

