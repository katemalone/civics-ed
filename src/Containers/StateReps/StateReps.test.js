import React from 'react';
import { shallow } from 'enzyme';
import { StateReps } from './StateReps';

describe('StateReps', () => {


  it('should match snapShot', () => {
    const wrapper = shallow(
      <StateReps />);
    expect(wrapper).toMatchSnapshot()
  });
});