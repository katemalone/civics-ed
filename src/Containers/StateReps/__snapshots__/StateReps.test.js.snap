// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StateReps should match snapShot 1`] = `
<section
  className="StateReps_card"
>
  <h3 />
  <p>
    Chamber: 
  </p>
  <p />
  <p>
    Contact info 
  </p>
  <img
    className="StateReps_img"
  />
</section>
`;
